src/ResourceManager/KeyVault/Commands.KeyVault/Commands/NewAzureKeyVault.cs

```diff
@@ -114,12 +114,12 @@ public override void ExecuteCmdlet()
                     // This is to unblock Key Vault in Fairfax as Graph has issues in this environment.
                     WriteWarning(ex.Message);
                 }
-                if (userObjectId != Guid.Empty)
+                if (!string.IsNullOrWhiteSpace(userObjectId))
                 {
                     accessPolicy = new AccessPolicyEntry()
                     {
                         TenantId = GetTenantId(),
-                        ObjectId = userObjectId,
+                        ObjectId = Guid.Parse(userObjectId),
```

Comment:
Yes. This is phase 1 of the change (the access policy entry ObjectId is still GUID). The phase 2 change will come once Azure SDK for .NET is modified to permit strings in Microsoft.Azure.Management.KeyVault.AccessPolicyEntry.